platform/database/database.go

```diff
@@ -209,6 +210,26 @@ func (db *prodDB) SelectContext(ctx context.Context, dest interface{}, statement
 	return err
 }
 
+// SelectMultipleContext fetches a slice of elements from database.
```

Comment:
Could add a bit more to the comment since it appears as documentation here ? I'd love to have the syntax we're supposed to use for the query: `SELECT  FROM WHERE id IN (?)".
Otherwise we have to dig into the code everytime we want to check the syntax.

Reply:
but that is just basic postgres syntax?

I will add a comment anyway

Reply:
Oh my bad, I've never used the parametric version of IN directly in psql, and I didn't know.